test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -75,4 +77,18 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
         List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
         assertEquals(String.join("\n", list1), String.join("\n", list2));
     }
+
+    /**
+     * Executes the command, and asserts the result message is as expected.
+     */
+    public static void assertCommandResult(Command command, AddressBook addressBook, AddressBook expectedAddressBook,
+                                                                                               String expectedMessage) {
```

Comment:
@zzzzwen 
> in this case, how to indent for the parameters

One method you can try is to split the argument list at a different location instead, e.g.
```java
    public static void assertCommandResult(Command command, AddressBook addressBook,
                                           AddressBook expectedAddressBook, String expectedMessage) {
```